_generated/HelmDeployV0/src/helm.ts

```diff
@@ -84,7 +84,11 @@ async function run() {
     var kubectlCli: kubernetescli;
     var externalAuth = connectionType === "None" && (command === "install" || command === "upgrade");
     if (externalAuth && !tl.getVariable("KUBECONFIG")) {
-        tl.error("KUBECONFIG kube configuration file path must be set when connectionType is none and command is install or upgrade.");
+        if (fs.existsSync("$HOME/.kube/config")) {
```

Comment:
I'm not sure about the $HOME expansion here.  (just double checking that it was tested+works)

Reply:
```
[root@DESKTOP-OA5AB39 ~]# ls -l $HOME/test.sh
-rwxr-xr-x 1 root root 156 Mar  3 16:00 /root/test.sh
[root@DESKTOP-OA5AB39 ~]# node
Welcome to Node.js v20.4.0.
Type ".help" for more information.
> fs = require('fs');
> fs.existsSync("$HOME/test.sh")
false
> fs.existsSync("/root/test.sh")
true
```

The variable expansion does not work.

Reply:
Updated and tested:

> fs = require('fs');
> path = require('path');
> const kubeConfigPath = path.join(process.env.HOME, '.kube', 'config');
undefined
> fs.existsSync(kubeConfigPath)
true

Reply:
@tkasparek  Could you take a look at it again? Thanks